### PR TITLE
book sort implemented

### PR DESCRIPTION
**Pull request by JelenaDinic**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function App() {
   const [ isLogged, setIsLogged ] = useState(false)
   const [ searchInput, setSearchInput ] = useState('')
   const [ filters, setFilters ] = useState<WhereObject[]>([])
+  const [ sorts, setSorts ] = useState<string[]>([])
```
Comment (brankastevic) [inline src/App.tsx:17]: Not so important, but 'sorts' is a confusing name. Maybe you can rename it `[ sorting, setSorting ] `

File: src/components/createBook/CreateBook.tsx [hunk 2]
```diff
@@ -141,8 +141,8 @@ function CreateBook() {
                 getOptionValue={(option: AuthorResponse) => option.Id.toString()}
                 onChange={handleSelectedAuthorsChange}
               />
-              <button className='add-btn' onClick={() => setShow((s) => !s)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
-              { show && <CreateAuthor setIsAuthorsChanged={setIsAuthorsChanged} closeModal={() => setShow(false)}/>}
+              <button className='add-btn' onClick={() => setShowModal((s) => !s)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
```
Comment (brankastevic) [inline src/components/createBook/CreateBook.tsx:144]: Avoid naming with one letter like `(s) => !s` 
While it doesn't make a huge difference, it reads better to me.

File: src/components/sort/Sort.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import Select, { MultiValue } from 'react-select';
+
+import './Sort.css';
+
+interface Props {
+    closeSortModal: () => void;
+    setSorts : React.Dispatch<React.SetStateAction<string[]>>
+  }
+
+  interface SortOption {
```
Comment (brankastevic) [inline src/components/sort/Sort.tsx:12]: Wrong indentation, format code please

File: src/components/sort/Sort.tsx
Comment (SaleBojke) [inline src/components/sort/Sort.tsx:1]: It would be good if you renamed this component, just Sort is a bit confusing.

**Review by SaleBojke — APPROVED** [review]

**Pull request merged**